packages/ui-test-utils/src/utils/expect.ts

```diff
@@ -31,6 +31,10 @@ import promised from 'chai-as-promised'
 import assertions from './assertions'
 
 const init = (chai: Chai.ChaiStatic) => {
+  // eslint-disable-next-line no-param-reassign
+  chai.config.truncateThreshold = 999
+  // eslint-disable-next-line no-param-reassign
+  chai.config.includeStack = true
```

Comment:
this is how I could configure chai. Now it wont truncate errors, so we will get nice errors instead of stuff like "abc has the value of a.... but expecteted true". It cuts off at 40 characters by default :/